tachyon/zk/expressions/sum_expression_unittest.cc

```diff
@@ -16,8 +15,8 @@ class SumExpressionTest : public math::FiniteFieldTest<F> {};
 TEST_F(SumExpressionTest, DegreeComplexity) {
   std::unique_ptr<ConstantExpression<F>> left =
       ConstantExpression<F>::CreateForTesting(F::One());
-  std::unique_ptr<SelectorExpression<F>> right =
-      SelectorExpression<F>::CreateForTesting(plonk::Selector::Simple(1));
+  std::unique_ptr<ConstantExpression<F>> right =
+      ConstantExpression<F>::CreateForTesting(F::One());
```

Comment:
These changes don't appear to adhere to the theme of the commit! Could you change these in a separate previous commit?

Reply:
I think they are because `SelectorExpression` is out of scope of base expression testing because we moved them to plonk namespace. What do you think?

Reply:
Hmm I suppose in that case, this change does seem to belong in this commit! I'll resolve.